airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```diff
@@ -72,7 +70,7 @@
       },
       "access_key_id": {
         "type": "string",
-        "description": "The access key id to access the S3 bucket. Airbyte requires Read and Write permissions to the given bucket.",
+        "description": "The access key id to access the S3 bucket. Airbyte requires Read and Write permissions to the given bucket, if not set, Airbyte will rely on Instance Profile",
```

Comment:
this should also mention that if acces_key_id is set then secret_key_id must also be provided

Reply:
done